cfgmgr/buffermgrd.cpp

```diff
@@ -21,25 +21,7 @@ using json = nlohmann::json;
 /* SELECT() function timeout retry time, in millisecond */
 #define SELECT_TIMEOUT 1000
 
-/*
- * Following global variables are defined here for the purpose of
- * using existing Orch class which is to be refactored soon to
- * eliminate the direct exposure of the global variables.
- *
- * Once Orch class refactoring is done, these global variables
- * should be removed from here.
- */
 int gBatchSize = 0;
```

Comment:
Can't we move this gBatchSize to recorder?

Reply:
No, won't be correct. gBatchSize is used for ConsumerStateTable under the Orch. It won't be correct to move gBatchSize into the Recorder Interface as they are not related

Reply:
But i think i can refactor Orch to remove the requirement of defining this global variables in the daemons.

Reply:
Handled